javascript-practice/src/js/app.js

```diff
@@ -12,11 +12,11 @@ export default class App {
 
     if (path.includes(PATHS.LOGIN)) {
       const view = new UserView();
-      const controller = new UserController(model, view);
+      const controller = new UserController();
       view.bindFormLogin(controller.handleFormLogin);
     } else if (path.includes(PATHS.DASHBOARD)) {
       const view = new DashboardView();
-      const controller = new DashboardController(model, view);
+      const controller = new DashboardController();
```

Comment:
All view initials should be inside controller. Controller is a place that we control the app follow. For current implement, we can not control app follow. Ping me if you can not catch an idea